boggs/src/main/java/com/hartwig/healthchecks/boggs/healthcheck/prestast/PrestatsHealthChecker.java

```diff
@@ -16,11 +17,11 @@
 	private static final String FAIL_ERROR = "FAIL";
 	private static final String FOUND_FAILS_MSG = "NOT OK: %s has status FAIL in file %s ";
 
-	private String runDirectory;
+	private final String runDirectory;
 
-	private PrestatsExtractor dataExtractor;
+	private final PrestatsExtractor dataExtractor;
 
-	public PrestatsHealthChecker(String runDirectory, PrestatsExtractor dataExtractor) {
+	public PrestatsHealthChecker(@NotNull String runDirectory, @NotNull PrestatsExtractor dataExtractor) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Line is longer than 80 characters (found 110).](https://www.codacy.com/app/hmf/health-checks/file/2780688150/issues/source?bid=3363458&fileBranchId=3365368#l24)